index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+<body>
+    <div class="menu_bar">
+        <div class="logo">My Logo</div>
+        <div class="menu_icon"><img src="images/Union.png"/></div>        
+    </div>
+    <div class="intro">
+        <div class="headline">
+            <main>
+                <h2>
+                     I’m May Pyone.
+                       Glad to see you!
+                </h2>
+                
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="connect">LET'S CONNECT</p>
+             <div class="connect_menu">
+                    <img src="images/twitter.png"/>
+                    <img src="images/linkin.png"/>
+                    <img src="images/M.png"/>
+                    <img src="images/git.png"/>
+                    <img src="images/Vector.jpg"/>
+             </div>
```

Comment:
- [ ] I kindly suggest that you update your code to make it more semantic and for the best HTML practices : the social icons should be listed inside a list `<ul> & <li>` instead of normal divs, and each social icon should be an `<a>` that contains the link for that icon
example:
 
 ```
 <ul class='...'>
 <li class='...'><a href='...' ><svg>...</svg></a></li>
 <li class='...'><a href='...' ><svg>...</svg></a></li>
 ...
 <li class='...'><a href='...' ><svg>...</svg></a></li>
 </ul>
 ```

Reply:
Thanks for your insightful feedback. Can I use img tag instead of svg?